docs/guide/models/alchemical_network.rst

```diff
@@ -29,12 +29,14 @@ information for each :class:`.ChemicalSystem`, the :class:`.Transformation`
 also contains a :class:`.Protocol` and, when relevant, atom mapping
 information for alchemical transformations.
 
-A :class:`.ChemicalSystem` is made up of one or more
-:class:`gufe.ChemicalComponent`\ s. Each component represents a conceptual
-part of the total molecular system, e.g., ligand, or protein, or solvent.
-This allows us to easily identify what is changing between two nodes -- for
-example, a RBFE edge for ligand binding should have the same solvent and
-protein components, but different ligand components.
+A :class:`.ChemicalSystem` is made up of one or more ``ChemicalComponent``
```

Comment:
This is what I meant by that? This is part of core models. I don't think it needs much more than this -- everything is linked to the API docs, so readers can click over there if they want more detail on any given object.

Reply:
I see, worth adding subheadings for faster navigation then?

Reply:
I don't subheads would be hard here -- you'd need a heading per paragraph for the last 3 paragraphs, and it would read strangely.

I do think some sort of visual representation of this information would be useful (at which point, the text is just there to clarify the image). But that's not happening in this PR.